packages/api/README.md

```diff
@@ -1,20 +1,36 @@
 # API
 
 <!-- toc -->
-- [Purpose and Vision](#Purpose-and-Vision)
-- [Package Lead](#Package-Lead)
-- [Roadmap](#Roadmap)
-- [Contributing](#Contributing)
-- [FAQ](#FAQ)
+  - [Purpose and Vision](#purpose-and-vision)
+  - [Package Lead](#package-lead)
+  - [Roadmap](#roadmap)
+    - [GraphQL serverless Function](#graphql-serverless-function)
+    - [Services layer](#services-layer)
+    - [Schema definitions](#schema-definitions)
+    - [Authentication](#authentication)
+  - [Contributing](#contributing)
+    - [Overview](#overview)
+    - [`importAll`](#importall)
+    - [`makeServices`](#makeservices)
+    - [`makeMergedSchema`](#makemergedschema)
+    - [`createGraphQLHandler`](#creategraphqlhandler)
+  - [FAQ](#faq)
+    - [How do I schema merge/stitch?](#how-do-i-schema-mergestitch)
+    - [What databases does Redwood support?](#what-databases-does-redwood-support)
+    - [Does Redwood Support SSR?](#does-redwood-support-ssr)
+    - [Can I use Redwood without an API/database?](#can-i-use-redwood-without-an-apidatabase)
 
 ## Purpose and Vision
 
-This package provides a GraphQL serverless Function with some defaults:
+Redwood believes the future is serverless and multi-client. And `@redwoodjs/api` makes Redwood serverless and multi-client ready. By exposing a single GraphQL endpoint, Redwood has one API to rule them all. Your API is abstracted away from any one side of your application, so you can have as many sides as you need, and when you need them.
 
-The GraphQL Function includes a Date, Datetime and JSON scalar types,
-the ability to merge schemas, a place to store resolvers in services, and authentication.
+Right now, the `@redwoodjs/api` package exposes functions that 1) build Redwood App’s serverless GraphQL API and 2) provide context to your services. The GraphQL API includes a Date, Datetime and JSON scalar types, the ability to merge schemas, a place to store resolvers in services, and authentication.
+
+Our vision is to provide frictionless and understandable integrations between the boundaries of Functions (capital F for serverless functions), GraphQL's Schema and Resolvers, Services, The Database, and Authentication.
+We also plan to make Functions platform agnostic. At the moment we're targeting AWS Lambda Functions, but we aim to provide a single interface for other providers, and built-time support when you've targeted them.
```

Comment:
Is it possible you meant `build-time` here?

Reply:
@tctrautman Very possible, thank you.

Reply:
@jtoar no problem at all.  Thank _you_ for your time spent making redwood more approachable.